docs/api/v1/fluxinstance.md

```diff
@@ -0,0 +1,466 @@
+# FluxInstance
+
+**FluxInstance** is a declarative API for the installation, configuration
+and automatic upgrade of the Flux distribution.
+
+A single custom resource of this kind can exist in a Kubernetes cluster
+with the name `flux` that must be created in the same namespace
+where the flux-operator is deployed.
+
+## Example
+
+The following example shows a FluxInstance custom resource that
+installs the upstream Flux distribution with all available components,
+and configures the flux-operator to automatically upgrade Flux 
+to the latest stable version:
+
+```yaml
+apiVersion: fluxcd.controlplane.io/v1
+kind: FluxInstance
+metadata:
+  name: flux
+  namespace: flux-system
+  annotations:
+    fluxcd.controlplane.io/reconcile: "enabled"
+    fluxcd.controlplane.io/reconcileEvery: "1h"
+    fluxcd.controlplane.io/reconcileTimeout: "3m"
+spec:
+  distribution:
+    version: "2.x"
+    registry: "ghcr.io/fluxcd"
+  components:
+    - source-controller
+    - kustomize-controller
+    - helm-controller
+    - notification-controller
+    - image-reflector-controller
+    - image-automation-controller
+  cluster:
+    type: kubernetes
+    multitenant: false
+    networkPolicy: true
+    domain: "cluster.local"
+  storage:
+    class: "standard"
+    size: "10Gi"
+  kustomize:
+    patches:
+      - target:
+          kind: Deployment
+          name: "(kustomize-controller|helm-controller)"
+        patch: |
+          - op: add
+            path: /spec/template/spec/containers/0/args/-
+            value: --concurrent=10
+          - op: add
+            path: /spec/template/spec/containers/0/args/-
+            value: --requeue-dependency=5s
+```
+
+You can run this example by saving the manifest into `fluxinstance.yaml`.
+
+1. Apply the resource on the cluster:
+
+   ```shell
+   kubectl apply -f fluxinstance.yaml
+   ```
+
+2. Run `kubectl get fluxinstance` to see the status of the resource:
+
+   ```console
+    $ kubectl -n flux-system get fluxinstance
+    NAME   AGE   READY   STATUS                           REVISION
+    flux   59s   True    Reconciliation finished in 52s   v2.3.0@sha256:4cc5babdb1279ad0177bf513292deadbfa3f7b7c3da0be7fa53b39ab434f7219
+   ```
+
+3. Run `kubectl describe fluxinstance` to see the reconciliation status components, conditions and events:
+
+    ```console
```

Comment:
```suggestion
```console
```

Reply:
The indentation is needed for the numbered list